spec/controllers/proposals_controller_spec.rb

```diff
@@ -173,6 +173,24 @@
           expect(query[2].id).to be(single.id)
         end
       end
+
+      it "returns JSON for preview count" do
+        login_as(user)
+        proposals = 3.times.map do |i|
```

Comment:
thanks, shortened. can't quite get rid of the block though, because we need to reindex each of the 3 proposals.